homework.md

```diff
@@ -4,110 +4,110 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+Red
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+HyperText Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+CSS used to define style for web pages including layout, design and variations
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
+h1 {
 
 }
-```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language commonly used in web development
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Help to share information
+Enable people to make their own changes
+Allow us to make checkpoints so that we can save our work as we go along
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy it
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+You download it to your computer
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+Pull request means submitting one's work so that someone can look into it and make possible changes
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+It is a History tab on GitHub Desktop whereas when you click you can see list of all commits thats been made in the repository
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+Push means to submit commits from your repository to GitHub 
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+A web service where you can sign up and get and account
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+It means to save your changes
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual Studio Code
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
```

Comment:
+ The commit command is used to save changes to a local repository after staging in Git